Formula/inform6.rb

```diff
@@ -20,6 +19,7 @@ class Inform6 < Formula
   end
 
   def install
+    ENV.deparallelize
```

Comment:
Can we report the need for this upstream?

Reply:
Apparently they're already aware: https://github.com/Homebrew/homebrew-core/pull/62648#issuecomment-764408732

I didn't realize @chenrui333 had previously tried to upgrade this package too; I discovered the deparallize issue independently, but found @DavidGriffith's comment while looking for a place to report the issue.

Reply:
Great; it looks like it won't be needed in the next version then. We should probably add a comment saying that.

Reply:
OK done